src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdminSearchPanel.cshtml

```diff
@@ -0,0 +1,41 @@
+@using System.Globalization
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using OrchardCore.Lists.ViewModels
+@using OrchardCore.Localization
+@model OrchardCore.Lists.ViewModels.ListPartViewModel
+@{
+    var dropdownClassList = new List<string>()
+    {
+        "dropdown-menu"
+    };
+
+    if (CultureInfo.CurrentUICulture.IsRightToLeft())
+    {
+        dropdownClassList.Add("dropdown-menu-start");
+    }
```

Comment:
create a string instead of building using a list. less resources and better performance.

Reply:
done